### DISCUSSION TITLE
HangMan game

### DISCUSSION THREAD

**Discussion by balogun14** [mcnaveen/Random-Words-API#102] [Ideas]
i made a game using your api this is the url https://hangman-game-dusky.vercel.app/

**Comment by mcnaveen** [comment]
Thanks for sharing. 🙏 

Due to high traffic and unexpected downtimes, I suggest you self-host this API.
This can be done simply by pressing the deploy button in the `Readme` file.

**Reply by balogun14** [reply]
okay thanks i will do that right away sir
